### PR TITLE
Add link to EUI and rework home page

### PR DESCRIPTION
**Pull request by florent-leborgne**
This PR changes a few things around the home page:

- Move the home page out of any category 
image

- Update the content of the home page 
image

- Add a link to the EUI docs 
image

### CODE REVIEW

File: docs/home.mdx [hunk 1]
```diff
@@ -1,22 +1,77 @@
...
+  }
+/>
 
+<DocRelatedArticles
+  sectionTitle='Writing copy and in-product help'
+  items={
+    [
+      {
+        "title": "UI Copy placeholder",
+        "description": "Terminology, resources, and tips for writing in and about the user interface.",
```
Comment (gchaps) [inline docs/home.mdx:61]: If I'm thinking of the right doc, then maybe:

```suggestion
        "description": "Terminology, resources, and tips for writing docs about the user interface.",
```

File: docs/home.mdx [hunk 2]
```diff
@@ -1,22 +1,77 @@
...
+        "pageId": "uiCopy"
+      },
+      {
+        "title": "In-product help",
+        "description": "Descriptions and examples of different types of in-product help.",
+        "pageId": "inProdHelp"
+      },
+       {
+        "title": "Writing for specific components",
```
Comment (gchaps) [inline docs/home.mdx:70]: Move this one to the first position.  How about listing what some of the components might be:

>**UI copy**
>Guidelines and examples for writing UI copy for for buttons, menus, error messages, and more.

File: docs/home.mdx [hunk 3]
```diff
@@ -1,22 +1,77 @@
...
+        "description": "Find writing guidelines for specific UI components directly in the EUI docs.",
+        "href": "https://eui.elastic.co",
```
Comment (gchaps) [inline docs/home.mdx:72]: Link to this page:  https://eui.elastic.co/#/guidelines/writing/guidelines

File: docs/home.mdx [hunk 4]
```diff
@@ -1,22 +1,77 @@
...
+      {
+        "title": "Docs placeholder",
+        "description": "Content types, minimum viable docs, tips and tricks for drafting technical content.",
+        "pageId": "techDocs"
+      },
+    ]
+  }
+/>
 
+<DocRelatedArticles
```
Comment (gchaps) [inline docs/home.mdx:55]: How about moving the copy section above the docs section?

File: docs/home.mdx [hunk 5]
```diff
@@ -1,22 +1,77 @@
...
-  and tone of the Elastic feature documentation, various content types, minimum 
-  viable docs, or tips and tricks for drafting technical content.
-* <DocLink id="uiCopy" />: if you are looking for information about terminology, 
-  tools, and tips for writing in and about the user interface.
+<DocRelatedArticles
+  sectionTitle='Writing docs'
+  items={
+    [
+      {
+        "title": "Docs placeholder",
```
Comment (gchaps) [inline docs/home.mdx:47]: We also have guidelines for deprecation messages that we could add here.

File: docs/home.mdx (hunk 3 above)
Comment (gchaps) [reply docs/home.mdx:72]: Screen Shot 2022-04-29 at 8 13 01 AM

Context help in Kibana > In-product help

File: docs/home.mdx [hunk 6]
```diff
@@ -1,22 +1,77 @@
...
```
[the remark this answers is not in the document]
Comment (florent-leborgne) [reply docs/home.mdx:47]: +1, placeholders were while waiting for additional pages to be pushed, I'll update with the pages we've moved to this repo already

File: docs/home.mdx [hunk 7]
```diff
@@ -1,22 +1,77 @@
...
-description: Home of Tech Docs guidelines.
-date: 2022-03-08
+title: Elastic Content Design Home
+description: Home of Tech Content guidelines.
+date: 2022-04-28
 tags: ['tech writing','ui copy', 'documentation']
 ---
 
-Welcome, Documentarian!
+Welcome, content designer!
```
[the remark this answers is not in the document]
Comment (florent-leborgne) [reply docs/home.mdx:10]: I'm trying with just bold to start with and avoid having too many titles. In fact, I'd like to shorten the intro text so the link sections are more visible.
Another option would be to use the title of the page, currently *Elastic Content Design Home* and tweak it a little.

File: docs/home.mdx [hunk 8]
```diff
@@ -1,22 +1,77 @@
...
+    ]
+  }
+/>
 
+<DocRelatedArticles
+  sectionTitle='Writing copy and in-product help'
+  items={
+    [
+      {
+        "title": "UI Copy placeholder",
```
[the remark this answers is not in the document]
Comment (florent-leborgne) [reply docs/home.mdx]: I had added placeholders while waiting for new pages to be merged. I'm updating those in the next commit.

File: docs/home.mdx (hunk 1 above)
Comment (florent-leborgne) [reply docs/home.mdx]: Now that István has included the page, I've used the description of that page directly.

File: docs/home.mdx (hunk 4 above)
Comment (florent-leborgne) [reply docs/home.mdx:53]: Good idea, doing it in the TOC as well

File: docs/home.mdx (hunk 5 above)
Comment (florent-leborgne) [reply docs/home.mdx]: 👍 István added them with his last merge, I'll pull the changes to this branch

**Comment by florent-leborgne** [comment]
@gchaps & @szabosteve, I've updated the PR based on the latest PRs merged by István and the feedback you provided, plus a few additional changes:

- "Writing about the UI" -> I've moved this section into the docs section

Some of the things that are still up in the air:

- The docs and UI copy sections start with pages that are for now empty, do we want to remove those or do you have plans for them @szabosteve? 
- We probably can shorten the intro of the home, or at least make a few tweaks so that readers's eyes get more quickly to the different link sections. 
- To be able to go live and publish to the internal next docs, we'll probably have to review the IDs of all our files and prefix them with something proper to our repo to avoid any duplicate in the wordlake-dev repo, that'll just grow and grow as teams get onboarded to the new system. maybe something as simple as `cd` (for content design)? 
- The tile that links to the EUI guidelines works, but it throws a non-blocking error on my local dev environment, I' will check this with the next-docs team.

File: docs/home.mdx [hunk 9]
```diff
@@ -1,23 +1,81 @@
...
+date: 2022-04-28
+tags: ["tech writing", "ui copy", "documentation"]
 ---
 
-Welcome, Documentarian!
+Welcome, **Content Designer**!
 
-This guide provides editorial guidelines for writing clear and consistent 
-documentation and UI copy for Elastic products.
+No matter if you are a tech writer, designer, developer, product manager, support
```
Comment (gchaps) [inline docs/home.mdx:12]: To shorten the intro, you could remove this paragraph.

**Comment by florent-leborgne** [comment]
Oops I pushed with a wrong account

**Comment by florent-leborgne** [comment]
I'm closing this PR and will open a clean one

**Pull request closed without merging**